code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -44,6 +44,13 @@
 		attack_delay = initial(attack_delay) - round(rage * 0.05,0.01) //-0.05 attack delay to a maximum reduction of -2.5
 	return ..()
 
+/mob/living/carbon/Xenomorph/Runner/handle_status_effects()
+	if(hit_and_run)
```

Comment:
Is there a possibility of this not being true?

Reply:
sanity check